site/src/pages/UsersPage/UsersPage.tsx

```diff
@@ -1,23 +1,10 @@
-import { User } from "api/typesGenerated";
-import { DeleteDialog } from "components/Dialogs/DeleteDialog/DeleteDialog";
-import { nonInitialPage } from "components/PaginationWidget/utils";
-import { useMe } from "hooks/useMe";
-import { usePermissions } from "hooks/usePermissions";
-import { FC, ReactNode, useState } from "react";
-import { Helmet } from "react-helmet-async";
-import { useSearchParams, useNavigate } from "react-router-dom";
-import { ConfirmDialog } from "components/Dialogs/ConfirmDialog/ConfirmDialog";
-import { ResetPasswordDialog } from "./ResetPasswordDialog";
-import { pageTitle } from "utils/page";
-import { UsersPageView } from "./UsersPageView";
-import { useStatusFilterMenu } from "./UsersFilter";
-import { useFilter } from "components/Filter/filter";
-import { useDashboard } from "components/Dashboard/DashboardProvider";
-import { useMutation, useQuery, useQueryClient } from "react-query";
+import { type FC, type ReactNode, useState } from "react";
```

Comment:
Almost all the diffs for this file are me just moving things around to group them better. The major things to look for are any code involving `groupsByUserIdQuery`